src/rust/engine/process_execution/src/lib.rs

```diff
@@ -441,6 +442,7 @@ impl CommandRunner for BoundedCommandRunner {
       .unwrap_or_else(|| "<Unnamed process>".to_string());
     let outer_metadata = WorkunitMetadata {
       desc: Some(desc.clone()),
+      level: Level::Info,
```

Comment:
This one should probably be `Debug`, since it means that the process hasn't actually started.